### ISSUE TITLE
Warning: You are setting the style `{ borderColor: ... }` as a prop.

### ISSUE THREAD

**Issue by omawang** [jeanverster/react-native-styled-toast#38]
Hi @jeanverster 

I get this warning every `toast` method was called.
```jsx
const { toast } = useToast()

const handleClick = () => {
  toast({
    message: 'Something wrong',
    intent: 'ERROR'
  })
}
```

Warning: You are setting the style `{ borderColor: ... }` as a prop. You should nest it in a style object. E.g. `{ style: { borderColor: ... } }`

App.js
```jsx
import React from 'react'
import { ToastProvider } from 'react-native-styled-toast'
import { ThemeProvider } from 'styled-components'

import AppContainer from './src/navigations/AppContainer'

const theme = {
  space: [0, 4, 8, 12, 16, 20, 24, 32, 40, 48],
  colors: {
    text: '#0A0A0A',
    background: '#FFF',
    border: '#FFF',
    muted: '#F0F1F3',
    success: '#7DBE31',
    error: '#FC0021',
    info: '#00FFFF',
  },
}

const App = () => {
  return (
    <ThemeProvider theme={theme}>
      <ToastProvider>
        <AppContainer />
      </ToastProvider>
    </ThemeProvider>
  )
}

export default App
```

package.json
```json
"react": "16.11.0",
"react-native": "0.61.5",
"react-native-styled-toast": "^1.0.25",
"styled-components": "^5.2.0",
"styled-system": "^5.1.5"
```

Thanks & regards

**Comment by omawang** [comment]
Fixed - after upgrading `react-native` version (in my case 0.63.3)
